Jenkinsfile

```diff
@@ -168,6 +168,10 @@ node('ibm-jenkins-slave-dind') {
         script: ". scripts/install-config.sh && echo \$CIZT_ZOWE_USER_DIR",
         returnStdout: true
       ).trim()
+      zoweKeystoreDir = sh(
+        script: ". scripts/install-config.sh && echo \$CIZT_ZOWE_KEYSTORE_DIR",
+        returnStdout: true
+      ).trim()
```

Comment:
We don't need this part since the variable is not useful in the remaining of Jenkinsfile. But it's ok.